pact.cabal

```diff
@@ -145,7 +145,6 @@ library
     Pact.Types.Command
     Pact.Types.Continuation
     Pact.Types.Crypto
-    Pact.Types.ECDSA
```

Comment:
Are we dropping this module because the WebAuthn support supercedes the functionality?

Reply:
While we were modifying the `Scheme` typeclass to support `webauthn`, another instance of that typeclass, the `ETH` scheme (and the `Pact.Types.ECDSA` module that supported it) came up as a place to eliminate some effectively-dead code.